release-notes.md

```diff
@@ -6,6 +6,10 @@
 - **Added capability to disable narrative production** - Added the `narrative` boolean option to allow users to disable narrative production. Locations, shape, length, and time are still returned. The narrative production is enabled by default.
 - **Added capability to mark a request with an id** - The `id` is returned with the response so a user could match to the corresponding request.
 
+## Release Date: 2016-01-18
+
+- **Data producer updates** - Data importer configuration (lua) updates to fix a bug where buses were not allowed on restricted lanes.  Ferry surface issue (Changed the default surface to be compacted for footways.).
```

Comment:
How about: "Fixed surface issue (change the default surface to be "compacted" for footways)."